config/default.yml

```diff
@@ -425,10 +425,11 @@ Style/EmptyLinesAroundModuleBody:
 # AutoCorrectEncodingComment must match the regex
 # /#.*coding\s?[:=]\s?(?:UTF|utf)-8/
 Style/Encoding:
-  EnforcedStyle: always
+  EnforcedStyle: never
```

Comment:
Since this cop is disabled by default, I assume that the default is that we live in a"post 1.9" world.
Therefore I thought `never` would be the more apropriate default.

However, this will cause offenses on existing configurations, when you just enabled this cop, without specifying an enforced style, e.g.:

````yml
Style/Encoding:
  Enabled: true
````